src/acom_music_box/main.py

```diff
@@ -152,6 +152,7 @@ def main():
     myBox = MusicBox()
     logger.debug(f"Configuration file = {musicBoxConfigFile}")
     myBox.readConditionsFromJson(musicBoxConfigFile)
+    myBox.check_config(os.path.join(os.getcwd(), musicBoxConfigFile))
```

Comment:
```suggestion
    myBox.readConditionsFromJson(musicBoxConfigFile)
```

Instead of having to call `check_config` here, I'd rather `readConditionsFromJson` call it internally and raise an exception